browser/ipfs/sources.gni

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import("//brave/components/ipfs/buildflags/buildflags.gni")
+import("//extensions/buildflags/buildflags.gni")
+
+brave_browser_ipfs_sources = []
+
+brave_browser_ipfs_deps = [ "//brave/components/ipfs/buildflags" ]
```

Comment:
As mentioned in https://github.com/brave/brave-core/pull/8561/files#r617404732, I think this dep doesn't belong here.